src/System-VMEvents/InputEventSensor.class.st

```diff
@@ -115,24 +115,26 @@ InputEventSensor class >> swapMouseButtons [
 
 { #category : #mouse }
 InputEventSensor >> anyButtonPressed [
-	"Answer whether at least one mouse button is currently being pressed."
+	"Answer whether at least one mouse button is currently being pressed.
+	 This can be checked by looking if any of the 3 bits representing the 3 mouse buttons states is set to 1."
 
-	^self mouseButtons anyMask: 7
+	^self mouseButtons anyMask: ButtonsBitMask
 ]
 
 { #category : #'modifier keys' }
 InputEventSensor >> anyModifierKeyPressed [
-	"ignore, however, the shift keys 'cause that's not REALLY a command key"
+	"Answer whether at least one modifier key is currently being pressed.
+	 This can be checked by looking if any of the 3 bits representing the state of cmd, opt and ctrl modifiers is set to 1."
 
-	^self modifiers anyMask: 16r0E	"cmd | opt | ctrl"
+	^self modifiers anyMask: ModifierKeysBitMask
 ]
 
 { #category : #mouse }
 InputEventSensor >> blueButtonPressed [
 	"Answer whether only the blue mouse button is being pressed. 
 	This is the third mouse button or cmd+click on the Mac."
 
-	^(self mouseButtons bitAnd: 7) = 1
```

Comment:
Here you can see the inconsistency with the original shared pool class variable value.